Gruntfile.js

```diff
@@ -287,10 +287,28 @@ module.exports = function (grunt) {
             all: [
                 '<%= meta.grunt %>',
                 '<%= meta.src %>',
-                '<%= meta.test %>'
+                '<%= meta.test %>',
+                //This code is third party, thus we should not touch it
```

Comment:
One comment is enough:

`// These modules include lots of third-party code, so we skip them`